hint.py

```diff
@@ -0,0 +1,57 @@
+import pygame
+import g
+import random
+import os
+import utils
+
+class Hint:
+    def __init__(self):
+        self.img = None
+        self.cxy = (780, 150)
+        self.ctry_idx = 0
+        self.hint = ''
+        self.ctry = ''
+        self.text = 'Click Show button to get hint!'
+        self.textxy = (530,500)
+
+    def get_country(self):
+        fname = os.path.join('data', 'countries.txt')
+        f = open(fname, 'r')
```

Comment:
You've opened a file and didn't close it, you can use a context manager so you wouldn't need to do it explicitly.